src/modules/clusters/cluster-list/cluster-list.js

```diff
@@ -364,6 +361,25 @@
                 cluster.isProfilingEnabled === "Unknown";
         }
 
+        function getTemplate(cluster) {
+            var tooltip = {
+                    "unmanagecluster": "If unmanage fails, resolve the issue and reinitiate unmanage cluster.",
+                    "importcluster": "If import cluster fails, resolve the issue before performing an unmanage cluster and reinitiate import.",
+                    "expandclusterwithdetectedpeers": "If cluster expansion fails, check if tendrl-ansible was executed successfully and ensure the node agents are correctly configured.",
+                    "ready": "The cluster is successfully imported for viewing monitoring data and metrics."
+                },
+                template = "";
+
+            if (cluster.currentStatus === "failed") {
+                template = tooltip[cluster.jobType.toLowerCase()];
+
+            } else if (cluster.readyState === true) {
```

Comment:
even if you don't equate the condition will work. 
else if (cluster.readyState){}

Reply:
Ahh yes :) My bad.